examples/advanced/rooInterface/Makefile.am

```diff
@@ -13,13 +21,17 @@ input_workspaces = \
 	WS_Poisson.root \
 	WS_Poisson_withSystematics.root
 
+CLEANFILES += *.txt
 $(input_workspaces): WS_%.root: PrepareWorkspace_%.C
-	$(ROOTEXEC) -l -b -q $<
+	$(ROOTEXEC) -l -x -q $<
 
-TEST_EXTENSIONS += .root
 ROOT_LOG_COMPILER = $(builddir)/runRooInterface
 
-# Disabled:
-# TESTS = $(input_workspaces)
+TESTS = $(input_workspaces) runBATCalculator.C
+
+# only works with root 5
+if WITH_CLING
+XFAIL_TESTS = runBATCalculator.C
```

Comment:
We should just leave it out, with ROOT-6, instead of adding it to `XFAIL_TESTS` - the latter carries the intention that the test *should* fail. Meaning, when it will pass, some day, because of other fixes, the build test as a whole would suddenly fail.